README.md

```diff
@@ -1,5 +1,21 @@
 # Automated headless authentication to OpenShift Cluster Manager
 
+## Installation
+
+Installation assumes that you already have Python3 on your machine.
+
+* Install [PDM](https://pdm-project.org/latest/#installation)
+* Install python dependencies
+    ```bash
+    $ pdm install
+    ```
+* Enter the virutal environment to run the rest of the commands:
+    ```bash
+    $ source .venv/bin/activate
+    [python3] $ playwright install chromium
+    [python3] $ python3 src/ocm_login/main.py --user <username> ...
```

Comment:
```suggestion
    $ pdm run playwright install chromium
    $ pdm run python src/ocm_login/main.py --user <username> ...
```
With `pdm`, you don't need to even activate the environment. Just run things from the `pdm` binary